docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: '3'
+
+services:
+  db:
```

Comment:
This one is conflicting with the postgres installed in my machine.
Would it make sense to use an alternative port for this?

Reply:
wat. how can something on docker conflict with your local machine? Is docker auto-exposing ports?

Reply:
hm yea, nvm, it doesn't make sense, i misread the error message (sorry, i had had too much wine yesterday)!
The problem I got was the first time i tried `docker-compose up` the database wasn't up yet, so Django app didn't start saying:

```
web_1  | django.db.utils.OperationalError: could not connect to server: Connection refused
web_1  | 	Is the server running on host "db" (172.18.0.2) and accepting
web_1  | 	TCP/IP connections on port 5432?
```

It worked the second time around, because by then the db was already up.

Maybe we should mention this in the README?

Or possibly install netcat on the image and do a bash loop to wait for the TCP port to be listening, like shown here: https://stackoverflow.com/a/31753726/149872

Reply:
This should be fixed now.